tests/unit/Serializers/ClaimSerializerTest.php

```diff
@@ -190,7 +190,7 @@ public function serializationProvider() {
 
 		$claim = new Statement( new Claim( new PropertyNoValueSnak( 42 ) ) );
 		$claim->setReferences( new ReferenceList( array(
-			new Reference()
+			new Reference( array( new PropertyNoValueSnak( 1 ) ) )
```

Comment:
This is a unit test for the ClaimSerializer, so it does not use the ReferenceSerializer. Instead, a simple fake is used.